dillo/web/posts/routes.py

```diff
@@ -326,17 +342,31 @@ def render_page():
             'max_results': '15'
             }, api=api)
 
-        # Populate the feed
-        for post in latest_posts._items:
-            author = post.user.full_name
-            updated = post._updated if post._updated else post._created
-            url = url_for('posts.view', post_shortcode=post.properties.shortcode, community_url=post.project)
-            content = post.properties.content[:500]
-            feed.add(post.name, str(content),
-                     content_type='html',
-                     author=author,
-                     url=url,
-                     updated=updated,
-                     published=post._created)
+        populate_feed(feed, latest_posts)
         return feed.get_response()
     return render_page()
+
+
+@blueprint.route('/p/feed/<community_url>/latest.atom')
```

Comment:
Proposed url change: `/c/<community_url>/feed/latest.atom` (for consistency)

Reply:
done, and done